src/Extension/FluentVersionedExtension.php

```diff
@@ -325,6 +325,83 @@ public function existsInLocale($locale = null)
         return $this->isDraftedInLocale($locale) || $this->isPublishedInLocale($locale);
     }
 
+    /**
+     * Check if this record has modifications in this locale
+     * Fluent friendly version of @see Versioned::stagesDiffer()
+     *
+     * @param string|null $locale
+     * @return bool
+     */
+    public function stagesDifferInLocale($locale = null)
```

Comment:
Good feature; Can I request that this be moved to RecordLocale object?

this makes this functionality available to all objects that have any fluent extension, not only fluentversionedextension. It also gives us the chance to consolidate some of the functionality that does record and locale inspection. :)

Reply:
It also means that you can check this field in template loops `<% loop $Locales %><% if StagesDifferInLocale %>...`

Reply:
This functionality is only relevant for versioned objects. You can still call this method on `RecordLocale` via `IsModified` on any object though ;-) I'll tidy up the naming so it's more clear.

Reply:
Fair enough, I accept that.

I will request a smaller change then; Can we use getRecordLocale() as the default (if $locale is not provided).

```php
$locale = $locale ?: $this->getRecordLocale();
```

That way my template support still works for looping over locales.

```html
<% loop $Locales %>
   <% if $Record.stagesDifferInLocale %>
    Modified in $Locale
  <% end_if %>
<% end_loop %>
```